README.md

```diff
@@ -1,8 +1,26 @@
 # MonotonicTickCount
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/monotonic_tick_count`. To experiment with that code, run `bin/console` for an interactive prompt.
+Implements a PORO that can be used for monotonic timestamping. It wraps a count of fractional seconds (or ticks) that can be initialized to the system monotonic clock via the `.now` method or to any float you supply. It implements the comparable interface and arithmetic operators for calculating offsets and differences.
 
-TODO: Delete this and the text above, and describe your gem
+For an explanation as to why this is preferrable to using the wall clock for timing calculations, see [this blog post](https://www.softwariness.com/articles/monotonic-clocks-windows-and-posix/).
+
+## Usage
+Comparing two monotonic timestamps
+```
+tick_count_a = MonotonicTickCount.now
+tick_count_b = MonotonicTickCount.now + 15.minutes
+tick_count_a < tick_count_b == true
+tick_count_b - tick_count_a == 900.0
+```
+
+Finding the elapsed tick count of a block
+```
+return_val, elapsed_ticks = MonotonicTickCount.timer do
+  sleep(10)
+  1
+end
+[return_val, elapsed_ticks] == [1, 10.0]
```

Comment:
I think this would be slightly more clear as two lines:
```
return_val        => 1
elapsed_seconds   => 10.0
```
The `=>` notation is pretty common in Ruby docs, to match what irb does on the next line. I'd change to that above as well.
And note that I changed the name to `elapsed_seconds`. Fortunately it's that, not ticks. ;)